src/Component/AutoMapper/Extractor/FromTargetMappingExtractor.php

```diff
@@ -139,4 +140,36 @@ private function transformType(string $source, Type $type = null): ?Type
             $this->transformType($source, $collectionValueTypes[0] ?? null)
         );
     }
+
+    /**
+     * PropertyInfoExtractor::isWritable() is not enough: we want to know if the property is readonly and writable from the constructor.
+     */
+    private function isWritable(string $target, string $property): bool
```

Comment:
this does not need to be ported into `SourceTargetMappingExtractor` because we don't use `PropertyInfoExtractor::isWritable()`